docs/variants/msi_z690/recovery.md

```diff
@@ -29,21 +29,29 @@ here.
   [Hardware connection / SPI](../development/#hardware-connection) section of
   the `Development` documentation.
 
-* Download official BIOS from vendor's website (this is the newest version, you
-  may choose an older one too or in the best case use your firmware backup):
+* If you [created a firmware backup before flashing to Dasharo](https://docs.dasharo.com/variants/msi_z690/initial-deployment/#flashing) you will be using it for this recovery process.
```

Comment:
@Firminator thank you for your contribution. We would appreciate wrapping at 80 characters. Best way to check your patch before sending is passing all pre-commit tests: https://github.com/Dasharo/docs#pre-commit-hooks

Reply:
[Here](https://results.pre-commit.ci/run/github/316342881/1673748147._jFaITS1S22Fs2vDSPFt0w) are the problems detected with your PR.

Reply:
All good now. I fixed it. Thx for letting me know.